{"Launchpad":"https://bugs.launchpad.net/bugs/1516765","Id":"1516765","Gerrit":"https://review.openstack.org/249239","IdGerrit":"249239","IdCommit":"ef1ccdaca9512b88878155f7d8c2c77853d91252","Files":["nova/tests/unit/virt/xenapi/test_volume_utils.py","nova/tests/unit/virt/xenapi/test_volumeops.py","nova/virt/xenapi/volume_utils.py","nova/virt/xenapi/volumeops.py"],"Lines_Inseted":7,"Lines_Deleted":2,"CommitParent":"fc932f1fbcf6199839c31918125d7fe775c4b5f6","Classification":"Bug","KeywordsTitle":" leak connection password, StorageError","Description":"storage error, password leaked,  not handled properly","KeywordsCommit":"storage error, so we dont have","Comments":"Password leakage fault "}